DEVDOC.md

```diff
@@ -313,3 +313,60 @@ A few useful homemade packages:
 ### Java
 
 > TODO
+
+## Code generated at compile time
+
+A big amount of code is generated at compile time. Code is typically generated
+via APIC (API Compiler, `cmd/apic/`). APIC takes as input `.api` and `.tmpl`
+files, and it generates code in another language like Go, C++ or Java.
+
+```
+
+               +----------+
+  foo.api  --->|          |
+               |   APIC   |---> foo.go (or foo.cpp, or foo.java, ...)
+  foo.tmpl --->|          |
+               +----------+
+
```

Comment:
This makes it look like the `.api`, `.tmpl`, and output files all need to be named the same. It also makes it sound like there can be only one of each. I would change it to:
`<.api files>`, `<.tmpl files>` for the input and `<generated files>` for the output.

(Since, technically, there is only one `.api` and one `.tmpl` file provided, and the "multiple" files part is done via imports, I'd accept `<.api file>` and `<.tmpl file>`, but for someone browsing the source this seems like a technicality, rather than representative of what it actually looks like).

Reply:
done